src/components/common/Containers.jsx

```diff
@@ -20,42 +20,60 @@ export const CardContainer = styled.div`
     /* Adjust width for smaller devices */
     max-width: 90%;
   }
-  `
-  
-  export const IconContainer = styled.div`
+`;
+export const Padding = styled.div`
```

Comment:
I think that this component should have a more specific name instead of just padding.